recommendations/nancy-chicago.md

```diff
@@ -0,0 +1,5 @@
+## Things to do in Riverwoods
```

Comment:
I didn't follow directions and I forgot to add my comment as a line comment - :smile: - **THIS** is a line comment!